quarkchain/cluster/master.py

```diff
@@ -1286,6 +1287,10 @@ def handle_new_root_block_header(self, header, peer):
         future_list = self.broadcast_rpc(
             op=ClusterOp.ADD_ROOT_BLOCK_REQUEST, req=AddRootBlockRequest(r_block, False)
         )
+
+        for grpc_slave in self.cluster_config.GRPC_SLAVE_LIST:
+            GrpcClient(grpc_slave.HOST, grpc_slave.PORT).set_rootchain_confirmed_block()
```

Comment:
we don't want to initialize a grpc client every time when we want to send a request. should have the clients initialized before.